x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -100,21 +58,18 @@ export function ServiceOverviewTransactionsTable(props: Props) {
     sort: DEFAULT_SORT,
   });
 
+  const { pageIndex, sort } = tableOptions;
+
+  const { transactionType } = useApmServiceContext();
   const {
-    data = {
-      totalItemCount: 0,
-      items: [],
-      tableOptions: {
-        pageIndex: 0,
-        sort: DEFAULT_SORT,
-      },
-    },
-    status,
-  } = useFetcher(() => {
+    uiFilters,
+    urlParams: { start, end, latencyAggregationType },
+  } = useUrlParams();
+
+  const { data = INITIAL_STATE, status } = useFetcher(() => {
```

Comment:
can you use the injected `callApmApi` here? That one is attached to the lifecycle of the component and automatically unborted if the component unmounts.